test/utilities/pageObjects/base-element.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const logger = require("../../config/logger-config");
+const EC = protractor.ExpectedConditions;
+
+
+class Element {
+    constructor(name, locator) {
+        this.name = name;
+        this.locator = locator;
+        this.element = element(locator);
+    }
+
+    TIMEOUT = 40000;
```

Comment:
const?

Reply:
Yes, I considered that variable was a const in the class. I've changed it to the usual variable. Is it correct now?

Reply:
yep